apps/openssl.c

```diff
@@ -469,6 +471,10 @@ int help_main(int argc, char **argv)
         }
     }
 
+    if (opt_num_rest() == 1) {
+        char *new_argv[] = { opt_rest()[0], "--help", NULL };
```

Comment:
blank line.  and I think this is more clear:
```
char *newargv[2];

newargv[0] = *opt_rest();
newargv[1] = "--help";
newargv[2] = NULL;

...
```
Do whatever, but I think dynamic-sized array initializers have been a problem on some platforms.

Reply:
It's actually fixed size, three elements.  But I do get complaints that initializing values dynamically isn't necessarely supported, so I still have to do it assignment style...